Model/src/BoardPackage/RailRoadSquare.java

```diff
@@ -1,7 +1,162 @@
 package BoardPackage;
 
-/**
- * Created by unclear on 2/5/18.
- */
-public class RailRoadSquare {
+import GamePackage.Player;
+
+/*****************************************************************
+ The RailRoadSquare class is a BoardSquare that tracks
+ information specific to RailRoadSquares.
+
+ @author Dylan Kernohan
+ @version 2/12/2018
+ *****************************************************************/
+public class RailRoadSquare extends BoardSquare{
+
+    /**The price of the RailRoad square*/
+    private final int PRICE;
```

Comment:
is it possible to call a constant and not instantiate it right away ? also can it even be instantiate in the constructor ?